### ISSUE TITLE
Font has changed in new version

### ISSUE THREAD

**Issue by omzy83** [yiisoft/yii2-gii#404]
I've just updated to version 2.1.0 and now the font looks horrible :(

![image](https://user-images.githubusercontent.com/8927010/54524137-0f623980-4969-11e9-8eaa-ae1b91e12b8d.png)

Looks like it is using Verdana:

![image](https://user-images.githubusercontent.com/8927010/54524119-040f0e00-4969-11e9-8f83-957e80d0c154.png)

From what I can see, Bootstrap 4 doesn't use Verdana as default, so not sure why this font is being used here. Same issue occurs in the Debug Toolbar.

**Comment by samdark** [comment]
What font would you prefer?

**Comment by omzy83** [comment]
The one used in the previous version was fine to be honest (it may have been Arial). Either way that Verdana font is really old school - I don't think anyone uses it anymore.

In Bootstrap 4 it looks like the font stack begins with the system fonts and then uses custom fonts after that. The font size looks a bit small once you remove Verdana, so may need to have a play with the font sizes as well.

**Comment by jkovacsab** [comment]
Related to this issue, I upgraded from 2.0.8 to 2.1.0 and noticed the font size shrank drastically (hard to read).
Also, the Diff pop-up lost its green and red line formatting, with only black text and gray rows, when I know there are changes and can see 'ins' tags available.
Can I do anything to bring back the previous formatting?  I only see a main.css loading, should there be any bootstrap css files?
In the 2.1.0 changelog, I see "Create (bootstrap)-independent version". Is that change related to these problems?

**Comment by samdark** [comment]
Yes. Bootstrap was upgraded to v4 and bundled with the extension itself.

**Comment by jkovacsab** [comment]
Thanks for the quick reply. I do see the Bootstrap css classes built into the main.css.  I deleted the specific web/assets folder, it regenerated, and that seemed to bring back the red and green diff lines. However, it did not help the excessively small [to me] body font-size of .75rem, coming from _reboot.scss. Why is that used, and am I missing something to get back to the default look and feel from 2.0.8?

**Comment by samdark** [comment]
Defaults changed in Bootstrap 4. We've managed to tweak most of the styles to look identical to what they were before but it seems not all of them.
